src/context/configContext.js

```diff
@@ -48,6 +48,12 @@ export const initialState = {
       zoom: '',
     },
   },
+  globalMessage: {
+    visible: false,
+    severity: 'info',
+    message:
+      ' Scheduled maintenance from 3:00am PST to 6:00am PST. (This is an example)',
+  },
```

Comment:
Great job!
I guess this will display on the app, right? Can we build a unit/integration test to demo this rather than put it into the real code?

Reply:
@andrewsu31098 my suggestion is: build a Notification.cy.js unit test for demonstration, in this way, we can cover the code by test, also, it is a handbook for people to use in the future, integrating into the layout is fine and easy, we can do it when needed.

Reply:
@andrewsu31098 do you need any help or guidance with this?